Config for standard networks

SiobhanPowell opened the pull request:
Made config class and a function in network_generation for working with the standard LV networks shipped with pandapower. Added lots of options for storage being added, not yet for the load feeds. Still need to test

SiobhanPowell committed:
Make config class and a function in network_generation for working with the standard LV networks shipped with pandapower. Added lots of options for storage being added, not yet for the load feeds. Still need to test

SiobhanPowell committed:
Tried to add the built in loads and sgen to the static_feeds dictionary, but dont know how to change the config from within the standard_lv file

SiobhanPowell committed:
Changed how static feeds is filled in in config file so that it includes the built in static generators and loads. Should this also include regular generators?

SiobhanPowell committed:
Resolved merge conflicts

SiobhanPowell committed:
Collected remote version of scratch notebook for resolving merge conflict fail

SiobhanPowell committed:
Testing and trouble shooting

bmeyers committed:
Update pp_network.py

small formatting thing

bmeyers commented on config.py:
```diff
@@ -50,6 +50,77 @@ def __init__(self, use_baseline):
+        if self.static_feeds_new is None:
+            self.static_feeds = {}
+        else:
+            self.static_feeds = self.static_feeds_new.copy()
```
I'm not following lines 81-93. It seems like a roundabout way to set `self.static_feeds = {}`.

SiobhanPowell committed:
Added new testing file

SiobhanPowell committed:
Changed how dictionaries are updated / added to to be nicer

SiobhanPowell committed:
Added capability to add new traditional generators in the config

SiobhanPowell committed:
Added some documentation and commenting

SiobhanPowell commented:
@bmeyers I added the option to add standard generators, changed the way dictionaries are added to (as per your suggestion), and added some documentation in the functions and methods. Let me know what you think!
